main/java/com/epam/utils/Utils.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.utils;
+
+import com.epam.dao.EntityDao;
+import com.epam.entity.UserAccount;
+import com.epam.exceptions.BusinessException;
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+/**
+ * Created by Almas_Doskozhin
+ * on 2/21/2016.
+ */
+public class Utils {
+    private static final Logger LOG = Logger.getLogger(Utils.class);
+
+    private Utils(){}
+
+    public static boolean checkUsers(){
+        try {
+            EntityDao entityDao = new EntityDao();
+            UserAccount user1 = entityDao.readUserAccount("user1");
+            UserAccount user2 = entityDao.readUserAccount("user2");
+
+            boolean user1Correct = user1.getDollarAmount().intValue()==30 &&
+                    user1.getEuroAmount().intValue()==136 &&
+                    user1.getRubleAmount().intValue()==1520 &&
+                    user1.getKZTAmount().intValue()==10000;
+            boolean user2Correct = user2.getDollarAmount().intValue()==126 &&
+                    user2.getEuroAmount().intValue()==130 &&
+                    user2.getRubleAmount().intValue()==780 &&
+                    user2.getKZTAmount().intValue()==25500;
+            return user1Correct && user2Correct;
+        }catch (IOException|BusinessException ex){
+            LOG.error(ex.getMessage(), ex);
```

Comment:
What for do you pass ex.getMessage? It will be logged as a part of stacktrace(your second param)